services/121-service/src/payments/fsp-integration/safaricom/interfaces/do-transfer.interface.ts

```diff
@@ -5,4 +5,5 @@ export interface DoTransferParams {
   readonly occasion: string;
   readonly originatorConversationId: string;
   readonly idNumber: string;
+  readonly transactionId: number;
```

Comment:
Rename transactionAmount to transferAmount.

Put attributes in a more logical order:
transactionId
transferAmount
phoneNumber
idNumber
remarks
occasion => add a note somewhere that the Safaricom API has misspelled this field: occassion. 

=> Order from "main stuff first" to "more details" and aligned with order elsewhere: SafaricomTransactionJobDto
Just to make it easier on the reader.

Reply:
Updated!